src/libs/actions/Task.js

```diff
@@ -569,19 +569,19 @@ function clearOutTaskInfoAndNavigate(reportID) {
  * Get the assignee data
  *
  * @param {Object} details
+ * @param {Object} personalDetails
  * @returns {Object}
  */
-function getAssignee(details) {
+function getAssignee(details, personalDetails) {
     if (!details) {
```

Comment:
Here we can get params `assigneeAccountID` instead of details & personalDetails.

Compute `details` here like this
 
 `details = personalDetails[assigneeAccountID];`